docs/releases/1.9.txt

```diff
@@ -239,6 +239,9 @@ Minor features
 * Added the :class:`~django.contrib.postgres.functions.TransactionNow` database
   function.
 
+* Fixed :class:`~django.contrib.postgres.forms.HStoreField` which has never
+  worked.
+
```

Comment:
Haven't tested myself but didn't it use to work as long as `has_changed` was not called?